azure/table_azure_cosmosdb_restorable_database_account.go

```diff
@@ -0,0 +1,172 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/cosmos-db/mgmt/2021-04-01-preview/documentdb"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureCosmosDBRestorableDatabaseAccount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_cosmosdb_restorable_database_account",
+		Description: "Azure Cosmos DB Restorable Database Account",
+		List: &plugin.ListConfig{
+			Hydrate: listCosmosDBRestorableDatabaseAccounts,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The friendly name that identifies the restorable database account.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "Contains ID to identify a restorable database account uniquely.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("ID"),
+			},
+			{
+				Name:        "account_name",
+				Description: "The name of the global database account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RestorableDatabaseAccountProperties.AccountName"),
+			},
+			{
+				Name:        "api_type",
+				Description: "The API type of the restorable database account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RestorableDatabaseAccountProperties.APIType"),
+			},
+			{
+				Name:        "creation_time",
+				Description: "The creation time of the restorable database account.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("RestorableDatabaseAccountProperties.CreationTime").Transform(convertDateToTime),
+			},
+			{
+				Name:        "deletion_time",
+				Description: "The time at which the restorable database account has been deleted.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("RestorableDatabaseAccountProperties.DeletionTime").Transform(convertDateToTime),
+			},
+			{
+				Name:        "type",
```

Comment:
Can we add a query that gives a count on the number of restorable database accounts per type?

Reply:
@misraved `type` is a common column across tables that shows us the resource type, which is always `Microsoft.DocumentDB/locations/restorableDatabaseAccounts` for this table.